### PR TITLE
Migrate to CheckoutV2Controller

### PR DESCRIPTION
**Pull request by stanislav-kozorez**

### CODE REVIEW

File: src/Feature/Checkout/client/Integration/Checkout/actionTypes.ts [hunk 1]
```diff
@@ -22,10 +22,10 @@ export const actionTypes = keyMirror({
   GET_BILLING_DATA_REQUEST: null,
   GET_BILLING_DATA_SUCCESS: null,
```
Comment (stanislav-kozorez) [inline src/Feature/Checkout/client/Integration/Checkout/actionTypes.ts:23]: Rename

File: src/Feature/Checkout/client/Integration/Checkout/tests/sagas.test.ts [hunk 2]
```diff
@@ -207,22 +206,22 @@ describe('submitFulfillmentStep', () => {
         };
         const gen = cloneableGenerator(sagas.submitFulfillmentStep)(payload.shipping);
 
-        test('should select checkoutShippingData', () => verifyShouldSelect(gen, selectors.checkoutShippingData, {}));
+        test('should select checkoutShippingInfo', () => verifyShouldSelect(gen, selectors.checkoutShippingInfo, {}));
         test('should throw exception, when shippingData was not loaded', () => {
```
Comment (stanislav-kozorez) [inline src/Feature/Checkout/client/Integration/Checkout/tests/sagas.test.ts:210]: Shipping info

File: src/Feature/Checkout/client/Integration/Checkout/tests/sagas.test.ts [hunk 3]
```diff
@@ -207,22 +206,22 @@ describe('submitFulfillmentStep', () => {
...
             } catch (e) {
-                expect((e as Error).message).toContain('ShippingData was not loaded');
+                expect((e as Error).message).toContain('ShippingInfo was not loaded');
             }
         });
 
-        test('should select checkoutDeliveryData', () => verifyShouldSelect(gen, selectors.checkoutDeliveryData, { data: { } }));
+        test('should select checkoutDeliveryInfo', () => verifyShouldSelect(gen, selectors.checkoutDeliveryInfo, { data: { } }));
         // tslint:disable-next-line
         test('should throw exception, when deliveryData was not loaded', () => {
```
Comment (stanislav-kozorez) [inline src/Feature/Checkout/client/Integration/Checkout/tests/sagas.test.ts:220]: deliveryInfo

File: src/scripts/generateTypescript.cake [hunk 4]
```diff
@@ -45,18 +45,34 @@ Func<TsGenerator> getGenerator = () => {
...
+	}
+	catch(ReflectionTypeLoadException loadException)
+	{
+		foreach(var e in loadException.LoaderExceptions)
+		{
+			Error(e.Message);
+		}
+        throw;
+	}
+	catch(Exception exception)
```
Comment (stanislav-kozorez) [inline src/scripts/generateTypescript.cake:73]: Format and change logic

File: src/Feature/Checkout/code/Infrastructure/Pipelines/Initialize/RegisterRoutes.cs [hunk 5]
```diff
@@ -37,20 +37,20 @@ private void RegisterHttpRoutes(RouteCollection routeCollection)
...
 
-            routeCollection.MapRoute(
-                nameof(CheckoutController),
-                Constants.CommerceRoutePrefix + $"/{CheckoutControllerName.ToLowerInvariant()}" + "/{action}",
-                namespaces: new[] { typeof(CheckoutController).Namespace },
-                defaults: new
-                {
-                    controller = CheckoutControllerName
-                });
+            //routeCollection.MapRoute(
```
Comment (larionovadaria) [inline src/Feature/Checkout/code/Infrastructure/Pipelines/Initialize/RegisterRoutes.cs:40]: To remove

Comment (stanislav-kozorez) [reply src/Feature/Checkout/code/Infrastructure/Pipelines/Initialize/RegisterRoutes.cs:40]: Already did in next draft MR

**Pull request merged**